src/create_disk.sh

```diff
@@ -184,7 +178,7 @@ case "$arch" in
         fi
         # shellcheck disable=SC2206
         sgdisk_args+=(-n ${BOOTPN}:0:+384M -c ${BOOTPN}:boot \
-                      -n ${ROOTPN}:0:"${rootfs_size}" -c ${ROOTPN}:root -t ${ROOTPN}:0FC63DAF-8483-4772-8E79-3D69D8477DE4)
+                      -n ${ROOTPN}:0:0 -c ${ROOTPN}:root -t ${ROOTPN}:0FC63DAF-8483-4772-8E79-3D69D8477DE4)
```

Comment:
Note this actually breaks Secure Execution. The 0 is greedy so takes all the remaining space, which leaves no space for the two partitions we need to create just after this line. We should be able to slightly adjust the offsets to still not have to require passing in the rootfs size. @jschintag is working on a fix.

Reply:
yeah. it appears `buildextend-secex` acts like `qemu`:

https://github.com/coreos/coreos-assembler/blob/e9b727824e98f2dfa0a8249afc5d89912f0bc47f/src/cmd-buildextend-metal#L19-L22

so it would have (previously, before this PR) had a 20G total disk size.. but wait..

does that mean secex artifacts have two partitions after the root partition? It means that the root partition would only ever be as large as the `rootfs_size` we were previously calculating and couldn't get grown on first boot?

Reply:
Actually, since this is breaking the RHCOS pipeline, I went ahead and opened https://github.com/coreos/coreos-assembler/pull/3697.

> does that mean secex artifacts have two partitions after the root partition? It means that the root partition would only ever be as large as the rootfs_size we were previously calculating and couldn't get grown on first boot?

Yeah, it has two partitions, but they're always nuked on first boot and we always reprovision the rootfs on LUKS: https://github.com/coreos/fedora-coreos-config/blob/206cc56fe0f0866f18f4c5c0fbd117dec77944ea/overlay.d/05core/usr/lib/dracut/modules.d/35coreos-ignition/01-secex.ign